src/panels/alarm-table/column_options.html

```diff
@@ -20,12 +20,12 @@ <h5 class="section-heading">Options</h5>
             <div class="gf-form-inline">
                 <div class="gf-form">
                     <label class="gf-form-label width-13">Apply to columns named</label>
-                    <input type="text" placeholder="Name or regex" class="gf-form-input width-13" ng-model="style.pattern" bs-tooltip="'Specify regex using /my.*regex/ syntax'" bs-typeahead="editor.getColumnNames" ng-blur="editor.render()" data-min-length=0 data-items=100 ng-model-onblur data-placement="right">
+                    <input type="text" placeholder="Name or regex" class="gf-form-input width-13" ng-model="style.pattern" ng-model-options='{ debounce: 250 }' bs-tooltip="'Specify regex using /my.*regex/ syntax'" bs-typeahead="editor.getColumnNames" ng-blur="editor.render()" data-min-length=0 data-items=100 data-placement="right">
```

Comment:
Can we move `{ debounce: 250 }` to a class in the stylesheet instead of repeating to for every input here?

Reply:
I didn't think you could put angular custom-processed stuff in stylesheets, it's gotta read it from the DOM.